packages/ibm-products/src/components/Datagrid/Datagrid.test.js

```diff
@@ -1981,6 +1984,41 @@ describe(componentName, () => {
       );
     });
   });
+  it('Customizing Columns disable save button when un-select all columns', async () => {
```

Comment:
Not necessarily for this PR, but it could be good to take a look at our conventions and naming for the base component and extensions. Could be good to come up with some better organization for the extensions themselves.